### PR TITLE
feat: use preconstruct to build `client` package and example usage in node

### PR DESCRIPTION
**Pull request by reecejohnson**

### CODE REVIEW

File: packages/client/package.json [hunk 1]
```diff
@@ -7,24 +7,15 @@
...
+  "main": "dist/lens-protocol-client.cjs.js",
+  "module": "dist/lens-protocol-client.esm.js",
   "private": true,
   "sideEffects": false,
   "files": [
     "dist/**/*"
   ],
   "scripts": {
-    "build": "pnpm clean && tsup",
-    "clean": "rimraf ./dist",
```
Comment (krzysu) [inline packages/client/package.json:27]: if you removed the `clean` script, do we still need `rimraf` as a dep here?

Comment (reecejohnson) [reply packages/client/package.json:27]: no we don't, cheers

**Review by krzysu — APPROVED** [review]
thanks!

**Comment by krzysu** [comment]
shall we add @babel/core as a dev dependency everywhere to resolve these? 

Screenshot 2023-02-16 at 10 37 55

**Pull request merged**